src/en/charms-resources.md

```diff
@@ -0,0 +1,103 @@
+Title: Resources
+TODO:  Critical: Improve examples
+
+# Juju Resources
+
+A Juju *resource* is additional content/files that a charm needs in order to
+run.
+
+There are three ways to examine resources:
```

Comment:
three types of resources?

Reply:
Not so simple. *Charm resources* are simply what resources a charm is defined with. Once the charm's application is running, you can look at the other two types, in a runtime fashion, from either a application or a unit standpoint. So they all seem intrinsically equivalent to me, hence "examining".

Reply:
ok